src/app/pages/users/tags/tags-list-table-data-source.ts

```diff
@@ -135,11 +142,35 @@ export class TagsListTableDataSource extends TableDataSource<Tag> {
     });
   }
 
+
+  public toggleRowSelection(row: Tag, checked: boolean) {
+    super.toggleRowSelection(row, checked);
+    this.deleteManyAction.disabled = this.selectedRows === 0;
```

Comment:
Enabling/Disabling a toolbar button according an item selection in the list should be done in a generic way by adding a new property in TableActionDef like 'linkedToListSelection: boolean' and use this prop in TableDataSource class to enable/disable automatically all these buttons.
You can also apply this change to all views in another issue.
Thanks!

Reply:
Done for this case, I will create an issue for the other cases.

Reply:
Not gut ;-)
Check my comments below